From ea45c790cc43a62bb78c42c11c1e45489e9753cf Mon Sep 17 00:00:00 2001
From: zhang sanshan <pete.zhang@nxp.com>
Date: Mon, 22 Jun 2020 10:50:19 +0800
Subject: [PATCH 34/40] MA-17422-1 genirq: Export irq_chip_pm_get and
 irq_chip_pm_put

Add EXPORT_SYMBOL_GPL entries for irq_chip_pm_get() and
and irq_chip_pm_put() so that we can allow drivers
like the gpio-mxc driver to be loadable as a module.

Signed-off-by: zhang sanshan <pete.zhang@nxp.com>
Reviewed-by: Fugang Duan <fugang.duan@nxp.com>
---
 kernel/irq/chip.c | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/kernel/irq/chip.c b/kernel/irq/chip.c
index 0b70811fd956..b3eb32140cc9 100644
--- a/kernel/irq/chip.c
+++ b/kernel/irq/chip.c
@@ -1606,6 +1606,7 @@ int irq_chip_pm_get(struct irq_data *data)
 
 	return 0;
 }
+EXPORT_SYMBOL_GPL(irq_chip_pm_get);
 
 /**
  * irq_chip_pm_put - Disable power for an IRQ chip
@@ -1624,3 +1625,4 @@ int irq_chip_pm_put(struct irq_data *data)
 
 	return (retval < 0) ? retval : 0;
 }
+EXPORT_SYMBOL_GPL(irq_chip_pm_put);
-- 
2.39.0

